kura/org.eclipse.kura.test/src/org/eclipse/kura/test/RemoteTargetTest.java

```diff
@@ -198,15 +198,15 @@ private void startingTests() {
         // hijack the settings
         try {
             Configuration mqttConfig = this.m_configAdmin
-                    .getConfiguration("org.eclipse.kura.core.data.transport.mqtt.MqttDataTransport");
+                    .getConfiguration("org.eclipse.kura.core.data.transport.mqtt.MqttDataTransport",  "?");
             Dictionary<String, Object> mqttProps = mqttConfig.getProperties();
             mqttProps.put("broker-url", "mqtt://broker-sandbox.everyware-cloud.com:1883/");
             mqttProps.put("topic.context.account-name", "EDC-KURA-CI");
             mqttProps.put("username", "EDC-KURA-CI");
             mqttProps.put("password", "PYtv3?s@");
             mqttConfig.update(mqttProps);
 
-            Configuration dataConfig = this.m_configAdmin.getConfiguration("org.eclipse.kura.data.DataService");
+            Configuration dataConfig = this.m_configAdmin.getConfiguration("org.eclipse.kura.data.DataService", null);
```

Comment:
FYI, Cristiano has cleaned up the test code in #568 . Probably okay to merge, but this will cause merge conflicts with that PR.